src/status_im/transport/inbox.cljs

```diff
@@ -179,6 +180,13 @@
 (def ^:private ^:const max-retries 10)
 (def ^:private ^:const retries-interval-change-threshold 3)
 
+(defn add-custom-mailservers [mailservers {:keys [db]}]
```

Comment:
I think we should do things differently and consider your new get-all-mailserver cofx to be the source of all mailservers, so when we create an account we save the default ones and there is no need to merge them everytime we login

Reply:
@yenda you mean storing the default mailservers in realm as well? 

I thought about that, but I decided against it, as it would make a bit harder changing them ( we would have to migrate realm data). Also you would have some code that stores them once on account creation, so it would not just be a matter of removing code and simplifying.

Personally it seems simpler to just merge the custom one on login, rather than making the default ones, less stateful. 

What do you think?